src/continuation/continuation_afam.jl

```diff
@@ -0,0 +1,107 @@
+        samples_per_parameter = 100, show_progress = true,
+    )
+    progress = ProgressMeter.Progress(length(prange);
+        desc = "Continuing attractors and basins:", enabled=show_progress
+    )
+    mapper = acam.mapper
+    reset_mapper!(mapper)
+    # first parameter is run in isolation, as it has no prior to seed from
+    set_parameter!(referenced_dynamical_system(mapper), pidx, prange[1])
+    fs = basins_fractions(mapper, ics; show_progress = false, N = samples_per_parameter)
+    # At each parmaeter `p`, a dictionary mapping attractor ID to fraction is created.
+    fractions_curves = [fs]
+    # The attractors are also stored (and are the primary output)
+    prev_attractors = deepcopy(extract_attractors(mapper))
+    attractors_info = [prev_attractors]
+    ProgressMeter.next!(progress; showvalues = [("previous parameter", prange[1]),])
+    # Continue loop over all remaining parameters
+    for p in prange[2:end]
+        set_parameter!(referenced_dynamical_system(mapper), pidx, p)
+        reset_mapper!(mapper)
+        # Seed initial conditions from previous attractors
+        # Notice that one of the things that happens here is some attractors have
+        # really small basins. We find them with the seeding process here, but the
+        # subsequent random sampling in `basins_fractions` doesn't. This leads to
+        # having keys in `mapper.bsn_nfo.attractors` that do not exist in the computed
+        # fractions. The fix is easy: we add the initial conditions mapped from
+        # seeding to the fractions using an internal argument.
+        seeded_fs = Dict{Int, Int}()
+        for att in values(prev_attractors)
+            for u0 in acam.seeding(att)
+                # We map the initial condition to an attractor, but we don't care
+                # about which attractor we go to. This is just so that the internal
+                # array of `AttractorsViaRecurrences` registers the attractors
+                label = mapper(u0; show_progress = false)
+                seeded_fs[label] = get(seeded_fs, label, 0) + 1
+            end
+        end
+        # Now perform basin fractions estimation as normal, utilizing found attractors
+        # (the function comes from attractor_mapping.jl)
+        fs = basins_fractions(mapper, ics;
+            additional_fs = seeded_fs, show_progress = false, N = samples_per_parameter
+        )
+        # We do not match attractors here; the matching is independent step done at the end
+        current_attractors = deepcopy(extract_attractors(mapper))
+        push!(fractions_curves, fs)
+        push!(attractors_info, current_attractors)
+        overwrite_dict!(prev_attractors, current_attractors)
+        ProgressMeter.next!(progress; showvalues = [("previous parameter", p),])
+    end
+    # Match attractors (and basins)
+    rmaps = match_continuation!(attractors, matcher)
+    match_continuation!(rmaps, fractions_curves)
```

Comment:
This function should be a low level function such as `replace_key_dict` or something similar.

Reply:
Which function should be low level? `match_continuation!`?

Reply:
The second call: 

`match_continuation!(rmaps, fractions_curves)`

It is just bookkeeping

Reply:
I have renamed the function to `match_sequentially!`, is this okay?